### PR TITLE
Bluetooth: Fixes for shell sample

### PR DESCRIPTION
**Pull request by Vudentz**
Some leftovers of the port to the new shell.

### CODE REVIEW

**Comment by jarz-nordic** [comment]
@Vudentz : Would you mind to rebase it to #10322 ?

**Review by sjanc — APPROVED** [review]

File: tests/bluetooth/shell/src/main.c [hunk 1]
```diff
@@ -86,21 +91,20 @@ static void cmd_hrs(const struct shell *shell, size_t argc, char **argv)
 		return;
 	}
 
-	shell_fprintf(shell, SHELL_ERROR, "%s:%s%s\r\n", argv[0],
-		      "unknown parameter: ", argv[1]);
+	error(shell, "%s:%s%s", argv[0], "unknown parameter: ", argv[1]);
```
Comment (jarz-nordic) [inline tests/bluetooth/shell/src/main.c]: to speed up code execution you could rephrase it like that:

`error(shell, "%s:%unknown parameter: %s", argv[0], argv[1]);`

**Comment by Vudentz** [comment]
@jarz-nordic done

File: tests/bluetooth/shell/src/main.c (hunk 1 above)
Comment (Vudentz) [reply tests/bluetooth/shell/src/main.c]: Yep, I was wondering about this when doing the initial code but then I forgot to mention why it was done like that in first place.

**Review by jakub-uC — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Comment by carlescufi** [comment]
@Vudentz CI failing

**Pull request merged**